README.md

```diff
@@ -38,11 +38,11 @@ optional arguments:
   -u [URL_COL], --url-col [URL_COL]
                         column with URLs to download (default: file_url)
   -w WAIT_TIME, --wait-time WAIT_TIME
-                        time to wait between tries (default: 3)
+                        seconds to wait between tries (default: 3)
```

Comment:
Is this retrying after failure, or waiting after a successful download before trying the next? The argument name suggests the latter, but I suspect it's the former? For the former, argument name should be more like `--retry-after`.

That being said, retry algorithms (at least robust ones) for internet protocols are normally written with an exponential escalation of wait time (such as 1, 2, 4, 8, 16, 32, .. seconds). In that case, a user may want to specify at which point to give up and log a failure, for example `--max-retries` and/or `--max-wait`.

Reply:
It's waiting after a failed attempt if the response is any of the following: 429, 500, 502, 503, 504. It doesn't have a wait between successful downloads. It has a max number of times to retry on the designated responses, but otherwise just logs the response in the error log (along with the index, filename, and url).

Setting a maximum wait time on a request would probably be a good idea as well. [urllib3.request](https://urllib3.readthedocs.io/en/latest/reference/urllib3.request.html) seems to handle much of this when also passed a [Retry object](https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#urllib3.util.Retry). @thompsonmj  had also suggested [HTTPAdapter](https://requests.readthedocs.io/en/latest/user/advanced/#transport-adapters) as an option that also uses `Retry`.

Reply:
Created issue #6 to address this.